sandman/source/schedule.h

```diff
@@ -1,8 +1,45 @@
 #pragma once
 
+#include <vector>
+
+#include "rapidjson/document.h"
+#include "rapidjson/filereadstream.h"
+#include "control.h"
+
 // Types
 //
 
+// A schedule event.
+struct ScheduleEvent
+{
+	// Read a schedule event from JSON. 
+	//
+	// p_EventObject:	The JSON object representing the event.
+	//	
+	// Returns:		True if the event was read successfully, false otherwise.
+	//
+	bool ReadFromJSON(rapidjson::Value::ConstObject const& p_EventObject);
+	
+	// Delay in seconds before this entry occurs (since the last).
+	unsigned int	m_DelaySec;
+	
+	// The control action to perform at the scheduled time.
+	ControlAction	m_ControlAction;
+};
+
+class Schedule {
```

Comment:
I don't think there is any code that needs to reference a schedule class, so in my opinion it's better to keep the declaration inside of the CPP file. 
Also the { should be on the next line.